src/api-server-selector.js

```diff
@@ -372,6 +377,16 @@ export class ApiServerSelector extends EventsTargetMixin(AmfHelperMixin(LitEleme
     this.selectIfNeeded();
   }
 
+  set selectedValue(value) {
```

Comment:
Weird, there's no comment I added earlier.
Why this change? What is this? How's this different from the `value`?